.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+***Exception or Stack Trace***
+Add the exception log and stack trace if available
+
+**To Reproduce**
+The arguments used to reproduce the behavior
```

Comment:
"arguments, steps, and/or code"

Reply:
Its just an exe tool - no steps or code here